app/store/actions/healthcareAuthorities/getHealthcareAuthoritiesAction.ts

```diff
@@ -0,0 +1,55 @@
+import { createAction } from '@reduxjs/toolkit';
+import Yaml from 'js-yaml';
+import { Dispatch } from 'redux';
+
+import { AUTHORITIES_LIST_URL_MVP1 } from '../../../constants/authorities';
+import { AppThunk } from '../../types';
+
+const GET_HEALTHCARE_AUTHORITIES_STARTED = 'GET_HEALTHCARE_AUTHORITIES_STARTED';
+const GET_HEALTHCARE_AUTHORITIES_SUCCESS = 'GET_HEALTHCARE_AUTHORITIES_SUCCESS';
+const GET_HEALTHCARE_AUTHORITIES_FAILURE = 'GET_HEALTHCARE_AUTHORITIES_FAILURE';
+
+const getHealthcareAuthorities_started = createAction(
```

Comment:
```suggestion
const getHealthcareAuthoritiesStarted = createAction(
```
What do you think about using camel case here?

Reply:
additionally its not clear to me what exactly 'started' means here.  What do you think about something like `userRequestedHealthcareAuthorities`?

Reply:
I'm not too opinionated about this so I'm down for changing it, but here's my rationale for this unconventional typing:

```
getHealthcareAuthoritiesStarted
getHealthcareAuthoritiesSuccess
getHealthcareAuthoritiesFailure
```
Is a lot of cognitive load to distinguish between the different actions. To me it's easier for a case like this where there are three variables intentionally similar to use `_` to denote where the distinction is. Kinda like using `_property` convention on a private property.

Reply:
Ah i see the reasoning now.  I too dont have a strong opinion, so if you prefer the getThing_started style i say go for it.

I guess im used to seeing actions named like:
```
userRequestedData
dataResponseReceived
```
so there's less duplication in the action names, but yeah if every network request had the three similarly named actions, I see the benefit of the unconventional casing.  (also what would you call this, camelSnake_case?)